ci/builders/linux_web_framework.json

```diff
@@ -0,0 +1,40 @@
+{
+    "builds": [
+        {
+            "archives": [
+                {
```

Comment:
I believe you can leave archives empty, or you may even be able to remove it. The whole build will be uploaded to cas and you can reference it in the test dependencies by name.

Reply:
So if I remove archives then I can reference the created output with the build name?

Reply:
Ah yeah you can reference it in the dependencies.